js/src/modal/modal.spec.js

```diff
@@ -602,6 +800,751 @@ describe('Modal', () => {
 
       modal.show()
     })
+
+    it('should not add compensation padding when body boundary width is less than window width with no vertical scrollbar before showing and no vertical scrollbar after showing', done => {
+      loadModalCss()
+      document.body.style.width = window.innerWidth - 1 + 'px'
+      document.body.style.height = '1px'
+      document.documentElement.style.paddingRight = null
+      unstyleFixture()
+
+      fixtureEl.innerHTML = getModalInnerHtml()
+
+      const modalEl = fixtureEl.querySelector('.modal')
+      const modal = new Modal(modalEl)
+
+      modalEl.addEventListener('show.bs.modal', e => {
```

Comment:
what's the interest of checking if the `e` is defined? Is it something you want to test?